packit/api.py

```diff
@@ -275,11 +275,14 @@ def sync_release(
 
             new_pr = None
             if create_pr:
-                new_pr = self.push_and_create_pr(
-                    pr_title=f"Update to upstream release {full_version}",
-                    pr_description=description,
-                    dist_git_branch=dist_git_branch,
-                )
+                title = f"Update to upstream release {full_version}"
+
+                if not self.dg.pr_exists(title, dist_git_branch):
```

Comment:
Is checking only the title enough? What if there is a change in the upstream repo and we want to propose a new update with the same release number but a different commit. What should be the workflow in this case?

Also: I think we should check only the open PRs.

Reply:
Right, I will add the check of description (it contains the tag and commit).

>  Also: I think we should check only the open PRs.

The used `get_pr_list()` method returns only opened PRs by default.

Reply:
> The used `get_pr_list()` method returns only opened PRs by default.

Uh, bummer! I've checked for this, but was totally blind to that default value :see_no_evil:

Reply:
We can explicitly set that value to make it really clear. Good point.

Reply:
> What if there is a change in the upstream repo and we want to propose a new update with the same release number but a different commit.

If I get this right, we already pushed the changes in this state so the existing PRs should be updated -- I personally like that behaviour.

Reply:
I added the check for the same description since if we not check the same description (which contains commit), no change will be pushed and PR will not be updated at all.